x-pack/plugins/observability_solution/observability/server/services/slo/get_preview_data.ts

```diff
@@ -73,8 +74,12 @@ export class GetPreviewData {
 
     const truncatedThreshold = Math.trunc(indicator.params.threshold * 1000);
 
+    const index = options.remoteName
```

Comment:
I'm pretty sure this isn't necessary. The APM indices are already fetched dynamically, and, if set by the user, would include the remote indices they want to include in their local SLOs.

Reply:
This is going to be a problem when the remote cluster is using CCS to query it's source data. For example, let's say they have 2 levels of overview clusters: the first is the at the top which CCS to the regional cluster which then remotes to where the soruce data is. We could end up with `remote: remote: index`.